testapp/Dockerfile

```diff
@@ -9,14 +20,14 @@ RUN meteor npm install && \
   npm install
 
 
-FROM node:8.15.1-jessie-slim
+FROM node:14.17.6-buster-slim
 
 # Install mongoDB client
-# The keys can get stale; to force the key import to re-run increment the "cache-bust" echo
-RUN echo "cache-bust 20190328" && apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
-RUN echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.6 main" > /etc/apt/sources.list.d/mongodb-org-3.6.list
+RUN apt-get update && apt-get install -y gnupg wget
+RUN wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | apt-key add -
+RUN echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/4.4 main" > /etc/apt/sources.list.d/mongodb-org-4.4.list
 
-RUN apt-get update && apt-get install -y mongodb-org-shell=3.6.11 netcat && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y mongodb-org-shell=4.4.8 netcat && rm -rf /var/lib/apt/lists/*
```

Comment:
Looks like this comes from here, and it seems like it might not be necessary with the official debian images:

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

```
In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.

Official Debian and Ubuntu images [automatically run apt-get clean](https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105), so explicit invocation is not required.
```